server/DB/schema.txt

```diff
@@ -33,6 +33,10 @@ CREATE TABLE users (
   password TEXT NOT NULL
 ) INHERITS (base);
 
+-- STATE POSSIBILITIES:
+-- false
```

Comment:
I don't know postgresql, so no idea what this does.  if this schema change is cool with Ian then you are g2g